INFO - 2023-09-25 02:44:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-25 02:44:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-25 02:45:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-25 02:46:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-25 02:47:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-25 02:47:23 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'juri_m' in C:\xampp\htdocs\html_tp1\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\html_tp1\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'juri_m', 3306, '', 0)
#1 C:\xampp\htdocs\html_tp1\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\html_tp1\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\html_tp1\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\html_tp1\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\html_tp1\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\html_tp1\app\Controllers\login_controller.php(26): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\html_tp1\system\CodeIgniter.php(932): App\Controllers\login_controller->auth()
#8 C:\xampp\htdocs\html_tp1\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login_controller))
#9 C:\xampp\htdocs\html_tp1\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\html_tp1\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'juri_m' in C:\xampp\htdocs\html_tp1\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\html_tp1\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\html_tp1\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\html_tp1\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\html_tp1\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\html_tp1\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\html_tp1\app\Controllers\login_controller.php(26): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\html_tp1\system\CodeIgniter.php(932): App\Controllers\login_controller->auth()
#7 C:\xampp\htdocs\html_tp1\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login_controller))
#8 C:\xampp\htdocs\html_tp1\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\html_tp1\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-25 02:47:23 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'juri_m'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\login_controller.php(26): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\login_controller->auth()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login_controller))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-25 02:48:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-25 02:48:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-25 02:48:24 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'juri_m' in C:\xampp\htdocs\html_tp1\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\html_tp1\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'juri_m', 3306, '', 0)
#1 C:\xampp\htdocs\html_tp1\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\html_tp1\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\html_tp1\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\html_tp1\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\html_tp1\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\html_tp1\app\Controllers\login_controller.php(26): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\html_tp1\system\CodeIgniter.php(932): App\Controllers\login_controller->auth()
#8 C:\xampp\htdocs\html_tp1\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login_controller))
#9 C:\xampp\htdocs\html_tp1\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\html_tp1\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'juri_m' in C:\xampp\htdocs\html_tp1\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\html_tp1\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\html_tp1\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\html_tp1\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\html_tp1\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\html_tp1\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\html_tp1\app\Controllers\login_controller.php(26): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\html_tp1\system\CodeIgniter.php(932): App\Controllers\login_controller->auth()
#7 C:\xampp\htdocs\html_tp1\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login_controller))
#8 C:\xampp\htdocs\html_tp1\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\html_tp1\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-25 02:48:24 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'juri_m'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\login_controller.php(26): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\login_controller->auth()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login_controller))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-25 02:48:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-25 02:48:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-25 02:48:46 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'juri_m' in C:\xampp\htdocs\html_tp1\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\html_tp1\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'juri_m', 3306, '', 0)
#1 C:\xampp\htdocs\html_tp1\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\html_tp1\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\html_tp1\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\html_tp1\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\html_tp1\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\html_tp1\app\Controllers\login_controller.php(26): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\html_tp1\system\CodeIgniter.php(932): App\Controllers\login_controller->auth()
#8 C:\xampp\htdocs\html_tp1\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login_controller))
#9 C:\xampp\htdocs\html_tp1\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\html_tp1\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'juri_m' in C:\xampp\htdocs\html_tp1\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\html_tp1\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\html_tp1\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\html_tp1\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\html_tp1\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\html_tp1\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\html_tp1\app\Controllers\login_controller.php(26): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\html_tp1\system\CodeIgniter.php(932): App\Controllers\login_controller->auth()
#7 C:\xampp\htdocs\html_tp1\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login_controller))
#8 C:\xampp\htdocs\html_tp1\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\html_tp1\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-25 02:48:46 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'juri_m'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\login_controller.php(26): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\login_controller->auth()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login_controller))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
